### ISSUE TITLE
[기능개선의견]레이아웃 info.xml 기본값 설정

### ISSUE THREAD

**Issue by eondcom** [xpressengine/xe-core#2389]
`<var name="layout_position" type="radio">
				<title xml:lang="ko">레이아웃 정렬</title>
				<title xml:lang="en">Layout position</title>
				<options value="center">
					<title xml:lang="ko">가운데 정렬　</title>
					<title xml:lang="en">Center justified　</title>
				</options>
				<options value="left">
					<title xml:lang="ko">왼쪽 정렬　</title>
					<title xml:lang="en">Left justified　</title>
				</options>
			</var>`

예를 들어 이런 레이아웃 설정 옵션이 있을 경우

`<option value="center" default>`

이렇게 설정해놓으면 미리 기본값으로 center값이 체크되어있으면 좋겠습니다.

**Comment by bjrambo** [comment]
```html
<!--@if(!$layout_info->layout_position)-->
center값
<!--@else-->
{$layout_info->layout_position}
<!--@end-->
```

템플릿에서 이런식으로 보통 기본값 활용하는 방법을 많이 사용합니다..
필요한거 있으면 if문으로 최대한 해결하고 option에서는 선택이 없을때 어떤값인지 ko상에서 알려주기만하면될거같아요 (어차피 그런거 다 class선언시 값없을때 center 값이 선택된거처럼 만들지 않나요 ㅎㅎ)

**Comment by eondcom** [comment]
네 저게 안되서 레이아웃 스킨상에서 하는데 사용자가 보기엔 직관적으로 어디에 체크되어있는지 보여주는게 낫죠^^;
코드 상에서 숨겨진걸 코드를 볼 줄 아는 사람이 아닌 경우를 대비해서 제안해봤습니다 ㅎ;

**Comment by kms0219kms** [comment]
본 XE1 버전의 개발은 작년 2월부터 사실상 중단 상태입니다.

이 문제가 계속된다면, XE1의 업데이트 버전인 [Rhymix](https://rhymix.org)으로 업데이트 해보시기 바랍니다.

업데이트 방법은 [여기](https://rhymix.org/manual/introduction/xe-upgrade)를 참고하시면 됩니다.

제대로 된 답변 드리지 못한 점 죄송합니다.
